internal/hcp/registry/types.builds.go

```diff
@@ -19,6 +19,7 @@ type Build struct {
 	Labels        map[string]string
 	Artifacts     map[string]packerSDKRegistry.Image
 	Status        hcpPackerModels.HashicorpCloudPacker20230101BuildStatus
+	Metadata      map[string]interface{}
```

Comment:
When we add phase 2 of metadata we'll have another parent key called CICD, which will get added, to make this code a bit more extensible for this case I would suggest making the Build model here with the same Metadata structure as the proto returns, where Metadata is an object that has a field called Packer which is a map[string]interface{}.  This means that when we add CI/CD you won't have to change any method signatures. What do you think?

Reply:
Done